.circleci/config.yml

```diff
@@ -1,51 +1,49 @@
-docker_image: &docker_image
-  image: circleci/ruby:2.7.2-node
-
```

Comment:
A lot of the changes in this file are slight renames of commands and jobs. The significant additions are the `rspec_system` command `system_specs` job.